experimenter/experimenter/jetstream/client.py

```diff
@@ -365,3 +369,21 @@ def get_population_sizing_data():
     sizing_data = get_sizing_data(suffix="latest")
     sizing = SampleSizes.parse_obj(sizing_data) if sizing_data is not None else {}
     return {"v1": sizing}
+
+
+def get_cached_population_sizing_data() -> SampleSizes | None:
```

Comment:
I'm not sure this should live here since it doesn't deal with Jetstream output or the celery tasks to retrieve it, and it's only used in `api/v5/types.py`. Any reason not to just put it in there?